storybook/__snapshots__/storyshots.test.js.snap

```diff
@@ -2033,7 +2033,7 @@ exports[`Storyshots ActivityFeedItem Collectible sent from KW to SW 1`] = `
                       ]
                     }
                   >
-                    label.sent
```

Comment:
This hints that maybe some logic has been inverted here from sent collectible -> received collectible. Please verify root cause here.

Reply:
It was old case when we were showing assets movement history between accounts and it was shown as "received" on one account and "sent" on other, however we no longer have this in place for key based so this is why it changes from `sent` (per story it was sent from KW) to `received ` because it was just an asset received to SW (Archanova wallet) and it's no longer marked as asset moving between accounts, hope this makes sense.